fetch_contigs.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+import urllib.request
+from Bio import Entrez
+
+def main(email=None, api_key=None, retmax=None):
+    Entrez.email = email
+    Entrez.api_key = api_key
+
+    handle = Entrez.esearch(db="assembly", term = "E. coli[Organism] AND contig[Assembly Level]", retmax = num)
+
+    record = Entrez.read(handle)
+    assembly_count = 0
+    for i in record['IdList']:
+        handle1 = Entrez.esummary(db="assembly", id = i)
+        record1 = Entrez.read(handle1)
+        url = record1['DocumentSummarySet']['DocumentSummary'][0]['FtpPath_GenBank']
+        file_name = os.path.basename(url)
+        link = os.path.join(url,file_name+'_genomic.fna.gz')
+        print(f'{file_name}.fna.gz' +" downloaded")
+        assembly_count += 1
+        urllib.request.urlretrieve(link, f'{file_name}.fna.gz')
+
+    print ("Total {} assemblies downloaded".format(assembly_count))            
+
+if __name__ == "__main__":

```

Comment:
If someone runs the scripts without any arguments you could print a help message showing the basic format of arguments that the script expects. i.e. `fetch_contigs.py NCBI_EMAIL NCBI_API_KEY NUM_CONTIGS`

Reply:
Added this below if __name__ == "main":

if len(sys.argv) != 6:
        print("Command usage: python fetch_contigs.py NCBI_EMAIL NCBI_API_KEY GENUS_NAME SPECIES_NAME NUM_CONTIGS")
        print("This script needs 5 arguments namely your email, your ncbi api_key, a genus and species name (e.g. Salmonella enterica) and the number of contigs you want to download. If you are not sure of how to obtain an NCBI API key, refer to the README document.")